src/styles/button.st.css

```diff
@@ -0,0 +1,131 @@
+/**
+ * = BUTTON - button.st.css
+ *
+ * You should have enough sizes and hooks
+ * but you can always add your own custom classes for whatever you need or 
+ * wrap this button up inside other buttons pre configured for your usecases.
+ * 
+ * ========================================================================= */
+
+
+/* = Default project vars. */
+:import {
+  -st-from: "@actionishope/shelley/styles/default/project.st.css";
+  -st-named:
+    --spacing-unit;
+}
+
+/* = Shelley project scope and vars. */
+:import {
+  -st-from: "./project.st.css";
+  -st-default: Shelley;
+  -st-named:
+    /* = CSS3 Vars. */
+    --color-ui,
+    --color-ui-inverse,
+    --color-ui-raw,
+    --color-ui-raw-inverse,
+    --color-accent-1,
+    --color-accent-2,
+    --color-accent-3,
+    --color-accent-4;
+}
+
+/* = Import Button CSS API */
+:import {
+  -st-from: "@actionishope/shelley/components/Button/button.st.css";
+  -st-default: Button;
+  -st-named:
+    --bg-color,
+    --border-color,
+    --text-color;
+}
+
+/* = Import ButtonGroup CSS API */
+:import {
+  -st-from: "@actionishope/shelley/components/ButtonGroup/buttonGroup.st.css";
+  -st-default: ButtonGroup;
+  -st-named: --seperator-color;
+}
+
+/* = This is how we can scope the styles in Stylable. */
+@st-scope Shelley {
+
+  /* = = The base <button> element. */
+  Button {
+    /* We'll assign this via our variants but set it with our tone prop values. */
+    --accent: initial;
+  }
+
+  /* = 'Text'  */
+  Button:variant(1) {
```

Comment:
Button is a bit bulky but it's mostly about setting up the button colours which are usually a bitch to override in frameworks. It's a trade of between a bit more theme work vs being locked in to a frameworks styles which we dislike.

All the colours are set in the project file.